src/Transactions.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react'
+import './Transactions.css'
 import transactionData from './data'
 
 export default function Transactions() {
+
     return (
-        <div>
+        <ul>
             {transactionData.map((data, key) => {
                 return (
-                    <div key={key}>
-                        {data.payee.shortname +
-                        " , " +
-                        data.date + 
-                        " , " +
-                        data.amount}
-                        </div>
+                  <li key={key}>
+                    <div className="logo">
+                      <img
+                        src={data.payee.imgsrc}
+                        alt="brandlogo"
```

Comment:
es wäre cool wenn du beim alt noch dazu schrieben könntest welche brand? wenn das jdm vorlesen lassen muss kann ers dann auch wirklich zu ordnen

Reply:
Ich wollte eigentlich da den Brand Namen einfügen, aber mit die JS Expression in den Anführungszeichen ging nicht... 

`alt="Brandlogo {data.payee.shortname}"`

aber ich habe es jetzt so gelöst:

`alt={data.payee.shortname}`